js/data/bucket.js

```diff
@@ -122,17 +123,17 @@ Bucket.prototype.populateBuffers = function() {
  * `vertexLength` vertices. If not, append a new elementGroup. Should be called
  * by `populateBuffers` and its callees.
  * @private
- * @param {string} programName the name of the program associated with the buffer that will receive the vertices
+ * @param {string} dataLayerTypeName type of buffer that will receive the vertices
```

Comment:
A bunch of things are already named *layer* and to me these aren't layers conceptually. I'm not sure I'd say that the icons and text within a single symbol layer are in different layers.  Maybe something like `dataBufferTypeName` or `dataInterfaceName`? this isn't *that* important.